dir_class.py

```diff
@@ -31,19 +32,22 @@ def get_control_string(self):
     def get_music_files_count(self):
         return len(self.music_files)
 
-    def move(self, destination_subdir, destination_dir):
-        for x, y, z in os.walk(current_dir):
+    def move(self, destination_subdir):
+        destination_dir = "zrobione"
+        for x, y, z in os.walk(self.base_dir):
             destination_pth = destination_dir + os.sep + destination_subdir
             os.makedirs(destination_pth, exist_ok=True)
             for f in z:
                 source_pth = x + os.sep + f
                 shutil.move(source_pth, destination_pth)
-            break
-        shutil.rmtree(current_dir)
```

Comment:
Usuwanie katalogu pozostaje tam gdzie było.

Nie jestem pewien czy to widzisz, ale wartości zmiennych `current_dir` i `self.base_dir` były sobie równe - to zawsze były ścieżki tych samych katalogów.

Reply:
> usuwa katalog do_zrobienia przed przeniesieniem wszystkich plików, tylko nie wiem co.

Kilka linijek wyżej ustawiasz `current_dir = "do_zrobienia"`, a w tej linijce usuwasz katalog którego ścieżka jest w zmiennej `current_dir`. Dlatego.